edge-detect/yolo/Dockerfile.old

```diff
@@ -0,0 +1,41 @@
+FROM iqtlabs/edgetech-core:latest
```

Comment:
@meadej is this file needed?

Reply:
No, unintentional commit. Addressed in https://github.com/IQTLabs/edgetech-coral-detect/pull/1.